### ISSUE TITLE
Increase tolerance for comparisons in 'make check' for ggttg (and add ggttg to the 'official' CI?)

### ISSUE THREAD

**Issue by valassi** [madgraph5/madgraph4gpu#571]
Thanks to @Jooorgen for reporting this: the `make check` test now fails in comparing check.exe and fcheck.exe for ggttg, becausethe difference is above 2E-4 relative for cross sections.

![image](https://user-images.githubusercontent.com/3473550/207816912-e77ef41c-d790-428a-93cf-a78cd21194b8.png)

This is in his modified version of the CI, where he added all processes. In the current 'official' CI instead now we are only running a couple of processes like eemumu and ggtt, but not ggttg. This is why we do not observe this in the CI.

This is also similar to the issues I reported in #542 for LHE file comparison. For some processes, the difference is above the percent level (second signifcant digit).

In general I think that we need to study numerical precision much better (especially going towards NLO). But I would separate for the moment two things
- For the software development process, it is important to have comparison tests fortran-cudacpp, and we can increase the tolerance even to some percent if required.
- For the physics production, at some point we need a scientific requirement on the precision needed. But this can come later, and will need differemt sets of studies.

In other words:
- for the moment, let's us larger tolerances and make tests succeed (while still being able to spot gross problems)
- eventually, let's study numerical precision much more in detail (also with a view to using float instead of double...)
